### PR TITLE
Decentralizes Robotics and Nerfs Mechs

### PR DESCRIPTION
**Pull request by AarontheIdiot**
## About The Pull Request

 This PR moves mech equipment to their respective department's Techfabs under a new 'exosuit equipment' category. Mech weaponry, medical exosuit equipment, most mining equipment, HONK mech equipment, and some engineering equipment are no longer available for production at mechfabs, and the mech missile rack has been removed from research and is no longer available for production, and is instead an item only spawnable by admins. Also makes some equipment a tad bit more expensive.

## Why It's Good For The Game

 Mechs are incredibly overpowered and unbalanced as of right now. Currently, the only thing that could stop a mech with a missile rack, disabler, and repair droid created by a traitor roboticist all from his little robo den is the hos's ion mode and some chump with the ion gun from the armory. That's a total of two people out of nearly 50 or more who are capable of quickly stopping murder mechs. The PR aims to balance out robotics, as well as decentralize it. If a department wants or needs a mech, they will ask robotics for it, robotics will make it and send it off to the department, and the department will equip it themselves for the use of the mech within the department, rather than all mech equipment being solely produced at ONLY robotics. In the game's current state, robotics could easily get their hands on lethal weaponry that outmatches weapons found in security's armory, which is why mech weapons have been moved to the security techfab. Missile racks have been removed due to the fact of their raw strength and potential to be abused, thus why they weren't moved to the security techfab.

## Changelog
:cl: AarontheIdiot
balance: Department mech equipment can no longer be produced at the mechfab and are now produced at their respective department lathes.
del: Removed Mech Missile Racks from RnD and Production, they are now an admin-spawn only item.
/:cl:

### CODE REVIEW

**Comment by FloranOtten** [comment]
Better version of the EMP PR, tbh. Though I would say keep a couple of basic weapons available, like the default laser weapon, the disabler, the grenade launcher (It launches flash bangs. Sec already has protection against them, they do no damage and it will probably stun the user as well) and such basic items in, for Blob usage.

**Comment by Tlaltecuhtli** [comment]
>move the guns 10 tiles away

lol

**Comment by AarontheIdiot** [comment]
> Though I would say keep a couple of basic weapons available, like the default laser weapon, the disabler, the grenade launcher (It launches flash bangs. Sec already has protection against them, they do no damage and it will probably stun the user as well) and such basic items in, for Blob usage.

i like this idea, editing now.

**Comment by FloranOtten** [comment]
> > move the guns 10 tiles away
> 
> lol

If you're as autistic as I am, you will have experienced how annoying it can be to nick the Security protolathe for usage in your collection of all the lathes.

**Comment by FloranOtten** [comment]
Also +1 purely because you're removing the damn missile rack, which was the most broken thing in the god damned universe. Seriously, a 3x3 stun and instant crit and delimit to whoever you hit

**Comment by Tlaltecuhtli** [comment]
what does this make it better than the other prs that got closed and did pretty much the same thing

**Comment by optimumtact** [comment]
who asked for this? who are you, what is going on? Is there a secret cave where you all discuss these pr's ?

**Comment by ishitbyabullet** [comment]
> Mech weaponry, medical exosuit equipment, most mining equipment, HONK mech equipment, and some engineering equipment are no longer available for production at mechfabs
That is dumb. The exofab requires you to remove materials from the protolathe material storage. I'm pretty sure there was a good reason for this even before departmental protolathes existed.

Also why the fuck should a roboticist go to medical bay or security so he can actually build a mech there? This is even more dumb if the roboticist could build the mech frame in robotics yet has to go to the service protolathe just to get the armor and equipment.

**Comment by zxaber** [comment]
This seems like an upgraded version of #44225, except we're doing it to all mechs and not just combat ones. Sure, it's the equipment this time and not the mech, but the result is pretty much the same. Honestly, it's weird to me that you don't want robotics printing _any_ non-combat mech equipment at all. I don't think there's a single other department that doesn't get to benefit from its own mid/end-game content, even if the content is primarily geared for others. Like, imagine if the chef couldn't eat his own food. Imagine if Botany wasn't able to use what they grow. Imagine if slime cores teleported to security when injected.

Rather than shatter mechs into a thousand little parts everywhere, we should add more counterplay that makes them fun (or at least possible) to fight against. Hence, my EMP PR, that disables equipment for a bit.

The missiles being removed, I don't have much issue with (except I thought my special handling of them in my mech ballistics PR was sorta cool). I fully intended (or intend, depending on how this PR ends) to make them do shit damage against mobs but high damage against walls and anchored objects (like other mechs). But removing the missiles isn't a big deal.

**Comment by Fikou** [comment]
> Honestly, it's weird to me that you don't want robotics printing _any_ non-combat mech equipment at all. I don't think there's a single other department that doesn't get to benefit from its own mid/end-game content, even if the content is primarily geared for others. Like, imagine if the chef couldn't eat his own food. Imagine if Botany wasn't able to use what they grow. Imagine if slime cores teleported to security when injected.

bro just leg actuator your way into security and get your weapons lol

**Comment by Arathianen** [comment]
This is overly harsh without any reason. It just kills mechs.

The other PR is way better and more thought out.

**Comment by CPTANT** [comment]
I kinda dislike this because it doesn't really leave anything for traitor roboticists.

**Comment by FloranOtten** [comment]
>This is overly harsh without any reason. It just kills mechs.
>The other PR is way better and more thought out.

>I kinda dislike this because it doesn't really leave anything for traitor roboticists.

And the other PR doesn't? You are unable to fight against ion spam. Traitor roboticists are more fucked that way. With this it takes a pinch of effort to get the weapons, instead of those weapons being made inept.

**Comment by imsxz** [comment]
> 
> 
> who asked for this? who are you, what is going on? Is there a secret cave where you all discuss these pr's ?

author is a sybil player

**Comment by dr-bee** [comment]
> 
> 
> I kinda dislike this because it doesn't really leave anything for traitor roboticists.

Building your own ghost cyborg army loyal only to you suddenly doesnt exist? Printing upload boards and subverting the AI not good enough for you? Traitor roboticists have many more options than murder mechs. If you cant circumvent your weapons being moved to a very steal-able lathe then you dont deserve murder mechs.

This isnt even a removal, this is a minor hurdle for a competent traitor with access to borgs to subvert.

**Comment by Dawson1917** [comment]
"Mechs are overpowered and there's no way to kill a mech without an ion gun!" 

Grab a toolbox. That's all it takes to kill every mech besides a gygax using yellow cells. If you think mechs are OP and not counterable then you haven't played with mechs very often at all. 
Also, arbitrarily moving robotics gear _out of robotics_ just makes roboticists have to break into departments every time they want a piece of mech equipment that doesn't fall under science's lathe, as well as giving an aneurysm to traitor roboticists who now have to break into sec in order to make any weapons.

Is this an i ded? I think it's an i ded

**Comment by imsxz** [comment]
side note this PR is lame and doesnt address any of mechs actual issues

issues being anything that isnt a humanoid trying to fight it, and gygax in general. it's pretty well accepted that mechs are easy to kill if you have 2 hands and a brain. the cancer of mechs shows most against simple mobs or immobile creatures like alien queens or blobs, gygax makes both of those fights an actual literal joke with a shitty punchline that got old the first few times.

this isnt really the way to go about it, i think mechs would be better off with a change that makes them need more involved in terms of maintenance and repairs and less random in terms of when it pops up. stuff that needs more than just spamming left click with a welder. Perhaps add something akin to replacing the armor plating which needs to be printed off at a mech fab.

tl;dr this change is dumb

**Comment by CRITAWAKETS** [comment]
Mech weapons are too strong so instead of nerfing them, you decide to make them slightly tougher to obtain.

Still doesnt change the fact that the weapons are too strong and instead it just adds an inconvenience that should not be there. Also, melee exists. Durand's punches can destroy r-walls and knock people out.

And then there is the gygax leg actuators, which is a top-tier weapon.

**Comment by datorangebottle** [comment]
Why are people so against fucking mechs in the ass with a rake? Validhunting roboticists need to die and stay dead. This is a good change, especially because it removes the most incredibly broken weapon from the station. This is exactly the gutting that mechs deserved.

So of course Oranges isn't going to merge it. There's no reason to discuss it any further.

**Comment by Arathianen** [comment]
>why are people against grudgecoding

**Comment by datorangebottle** [comment]
> > why are people against balance

if you think mechs are okay in their current state, you are part of the problem.

**Comment by Arathianen** [comment]
I guess I am part of the problem then.

The other PR which also nerfed mechs was actually good and balanced. This is just shit.

**Comment by HomocidlePancakes** [comment]
I don't get the point of this PR. It wants to nerf combat mechs by "decentralizing" robotics, but it just fails miserably. Does this discourage robotics from building mechs aimed towards helping the crew? Yes. Will this prevent robotics in the slightest from building their le-epic powergaming machines? No, and if it doesn't, then whats the fucking point? EMP PR did this better.

**Comment by GuyonBroadway** [comment]
I remember when ss13 was an RPG and not a competitive twitch-streamer game that needed mathematical, mechanical balance and counterplay...

**Comment by HomocidlePancakes** [comment]
You forgot to add "civil rights simulator"

**Comment by zxaber** [comment]
@datorangebottle 
>Why are people so against fucking mechs in the ass with a rake? Validhunting roboticists need to die and stay dead. This is a good change, especially because it removes the most incredibly broken weapon from the station. This is exactly the gutting that mechs deserved.

The PR that moved combat mechs out of Robotic's hands got turned down, so this one does the same effect but tries to hide it by also moving non-combat equipment away.

Mining is dead and you want to go do their job? First you gotta go bug the QM and hope he isn't a twat. Want to make Odysseus and use it to walk around and heal people? Oops, you're not allowed, drop it off at Medical and go back to your corner. Want to make a mech to try fight all these plasma fires? Jokes on you, that's Engineering's job, just sit in your no-equipment Firefighter and watch the station burn.

**Comment by ExcessiveUseOfCobblestone** [comment]
would have been funnier if you just tried to remove mechs completely tbh.

**Comment by AdamElTablawy** [comment]
there are nerfs that are unfun but need to happen

and then there's this

come on, at this point you might as well remove mechs completely. the other Pr does it better.

**Comment by 4dplanner** [comment]
> "Mechs are overpowered and there's no way to kill a mech without an ion gun!"
> 
> Grab a toolbox. That's all it takes to kill every mech besides a gygax using yellow cells.

made me lol

**Comment by Tlaltecuhtli** [comment]
if you have stimulus + dna vault half click cooldown + heal nanites + some general healing chems you can probably toolbox a gygax to death

**Comment by Shadowflame909** [comment]
3 assistants can toolbox a lesser ash drake to death.

You just have to be willing to make a sacrifice players are usually not willing to. Keep hitting until you die.

One guy got gibbed, and I died of burn damage shortly after.

Tis is a very possible endeavor though.

**Comment by necromanceranne** [comment]
I actually trust zxabers changes to be what would be a healthier change than these. Decentralizing mech equipment is just kind of like saying 'no toys for you'. Hidden Weapon Compartment also exists with the assumption you can print these weapons but that is practically impossible to make use of if they're decentralized. So that's a dead item.

Also Gygax gives no fucks. It is the odd duck for mechs by a far margain in that it isn't just the fast mech but the melee mech. You go fast and slap-fu anyone in your way to the floor.

**Comment by necromanceranne** [comment]
Also moving even basic mech equipment all over the place is just annoying. Sometimes as a robo I just wanna make a mining mech so I can personnally extract minerals out of lavaland because the miners died/aren't getting mats. I don't wanna have to bitch at the problem department to give me the means to do so when they are already presenting problems with getting mats. Not being even able to do that means nobody is geting shit because I can't access the tools I need and thus what do I do, tide in? More often than not you can just use the aux base to mine and not bother cargo at all until you deposit.

**Comment by Neo-0** [comment]
lets remove roboticist job

noice

**Comment by optimumtact** [comment]
datorangebottle: this nerf is good and needs to happen
me: merges cloning changes
datorangebottle: You are a shitter I hate you you are ruining my game why are you even born

**Comment by ExcessiveUseOfCobblestone** [comment]
Reminder that nearly every science job is balanced around you watching them to make sure they don't make their super powered death machine.

really jogs the nog.

also I don't see this PR going anywhere.

**Comment by ExcessiveUseOfCobblestone** [comment]
I'll reopen it if that other one gets closed if you remind me actually (but be prepared for mega changes)

**Comment by MegaEmpirical** [comment]
> Also moving even basic mech equipment all over the place is just annoying. Sometimes as a robo I just wanna make a mining mech so I can personnally extract minerals out of lavaland because the miners died/aren't getting mats. I don't wanna have to bitch at the problem department to give me the means to do so when they are already presenting problems with getting mats. Not being even able to do that means nobody is geting shit because I can't access the tools I need and thus what do I do, tide in? More often than not you can just use the aux base to mine and not bother cargo at all until you deposit.
> 
> Miners don't use the mechs. They will never use the mechs. We have a massive pushback against making them appealing miners for use. They are only used by roboticists sick of waiting for materials from lazy miners or selfish ones (getting tendrils/megafauna to the exclusion of material gathering). So why ever bother?

You would literally pass the lathe you need for that equipment just to get to mining.

**Comment by Trilbyspaceclone** [comment]
EMP scatter shot with warden Compack Shotgun can do a number to any mech...
